src/main/java/org/ditang/relaxng/defaults/RelaxDefaultsParserConfiguration.java

```diff
@@ -163,9 +163,8 @@ protected void insertRelaxDefaultsComponent() {
   public void setFeature(String featureId, boolean state) throws XMLConfigurationException {
     if (XERCES_SCHEMA_VALIDATION.equals(featureId)) {
```

Comment:
The RNG support reuses the `http://apache.org/xml/features/validation/schema` XSD schema validation feature and fails to pass the feature value to parent configuration. In the future, we might want to consider using some RNG specific feature URI to make it more explicit.

Reply:
Yes, I used that as a convenience to avoid defining yet another feature.
My PR here also has an automated test: https://github.com/dita-ot/dita-ot/pull/4238